src/app/components/shared/datetime-formats/datetime/abstract-datetime.component.ts

```diff
@@ -40,9 +46,9 @@ export abstract class AbstractDatetimeComponent extends AbstractTemplateComponen
 
   public update(): void {
     const zone = this.extractTimezone();
-    const valueInZone = this.value.setZone(zone);
+    const valueInZone = this.value().setZone(zone);
 
-    const timezoneName = AbstractDatetimeComponent.formatTimezone(zone, this.value);
+    const timezoneName = AbstractDatetimeComponent.formatTimezone(zone, this.value());
```

Comment:
we should pull `value()` out once and store in a variable

Reply:
chosen variable name: local variable `const value`

usage:

```ts
const timezoneName = AbstractDatetimeComponent.formatTimezone(zone, value);
```